vstools/utils/mime.py

```diff
@@ -194,9 +216,11 @@ def parse(
         return ParsedFile(filename, ext, encoding, file_type, mime)
 
     def is_index(self) -> TypeGuard[FileTypeIndexWithType]:
+        """Verify whether the result is of the ``Index`` type."""
```

Comment:
not the result, but the object is a complete INDEX FileType that holds a FileType, like one you'd find in a ParsedFile

FileType.VIDEO.is_index => False
FileType.INDEX.is_index => False
FileType.INDEX(FileType.VIDEO) => True, it's `index/video` mimetype